### PR TITLE
Update version matching matrix

### PR DESCRIPTION
**Pull request by stoshiya**

### CODE REVIEW

**Comment by stoshiya** [comment]
I've signed the CLA after this pull request.

File: README.md [hunk 1]
```diff
@@ -14,11 +14,12 @@ You need to install a version matching your Elasticsearch version:
...
 | master        |  Build from source          | See below  |
-| es-1.x        |  Build from source          | [2.6.0-SNAPSHOT](https://github.com/elasticsearch/elasticsearch-analysis-kuromoji/tree/es-1.x/#version-260-snapshot-for-elasticsearch-1x)  |
-|    es-1.5              |     2.5.0         | [2.5.0](https://github.com/elastic/elasticsearch-analysis-kuromoji/tree/v2.5.0/#version-250-for-elasticsearch-15)                  |
-|    es-1.4              |     2.4.3         | [2.4.3](https://github.com/elasticsearch/elasticsearch-analysis-kuromoji/tree/v2.4.3/#version-243-for-elasticsearch-14)                  |
-| < 1.4.5       |  2.4.2                      | [2.4.2](https://github.com/elasticsearch/elasticsearch-analysis-kuromoji/tree/v2.4.2/#version-242-for-elasticsearch-14)                  |
-| < 1.4.3       |  2.4.1                      | [2.4.1](https://github.com/elasticsearch/elasticsearch-analysis-kuromoji/tree/v2.4.1/#version-241-for-elasticsearch-14)                  |
+| es-1.x        |  Build from source          | [2.6.0-SNAPSHOT](https://github.com/elastic/elasticsearch-analysis-kuromoji/tree/es-1.x/#version-260-snapshot-for-elasticsearch-1x)  |
```
Comment (dadoonet) [inline README.md]: Can you change 2.6.0-SNAPSHOT TO 2.7.0-SNAPHOT here (and 260 by 270)?

**Comment by dadoonet** [comment]
Thanks. I need to fix other plugins as well. 
Left a small comment. Can you fix it?

File: README.md (hunk 1 above)
Comment (stoshiya) [inline README.md]: I changed at 26caf4a966c61f294a2e7cc1b3ab76653319addd.

**Comment by dadoonet** [comment]
Thanks!

**Pull request merged**